Casks/p/prism.rb

```diff
@@ -8,23 +8,28 @@
   homepage "https://graphpad.com/"
 
   livecheck do
```

Comment:
```suggestion
  # The `osVersion` parameter is required but doesn't seem to have an effect on
  # the version in the appcast. However, we may want to monitor this over time
  # (e.g. when the newest macOS release is higher than the hardcoded version).
  livecheck do
```

I'm not sure how a hardcoded macOS version will hold up over time (i.e., it could hypothetically cause a problem sometime in the future) but the `osVersion` value doesn't seem to have an effect right now, so long as something is provided. From what I've seen, the server always returns the same version (10.1.1) regardless of the `osVersion` value (`null`, `1`, `10`, etc.).

I'm okay with the hardcoded `osVersion` value for now (we can always revisit it in the future) but we may want to add an explanatory comment.